### PR TITLE
{Mysql} `az mysql flexible-server import create`: Update username value to be always null

### PR DESCRIPTION
**Pull request by nitishsharma25**
**Related command**

az mysql flexible-server import create

**Description**

This PR adds changes to make sure username is always null in the API. The username is fetched from the single server in the backend.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Mysql

File: src/azure-cli/azure/cli/command_modules/mysql/_validators.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def _mysql_import_data_source_type_validator(data_source_type, data_source_backu
 
 
 def _mysql_import_mode_validator(mode):
-    allowed_values = ['offline']
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/mysql/_validators.py:153]: This seems a breaking change. Does customer need to be aware?

Comment (nitishsharma2825) [reply src/azure-cli/azure/cli/command_modules/mysql/_validators.py:153]: It is a new feature and in the last PR, it was included in release doc so it should be fine

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**